solidity/deploy/09_transfer_proxy_admin_ownership.ts

```diff
@@ -0,0 +1,23 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
+  const { ethers, getNamedAccounts, upgrades, deployments } = hre
+  const { keepTechnicalWalletTeam } = await getNamedAccounts()
+  const { deployer } = await ethers.getNamedSigners()
+
+  // TODO: Once a DAO is established we want to switch to ProxyAdminWithDeputy and
+  // use the DAO as the proxy admin owner and ESDM as the deputy. Until then we
+  // use governance as the owner of ProxyAdmin contract.
+  const newProxyAdminOwner = keepTechnicalWalletTeam
+
+  deployments.log(`transferring ProxyAdmin ownership to ${newProxyAdminOwner}`)
+
+  const proxyAdmin = await upgrades.admin.getInstance()
+  await proxyAdmin.connect(deployer).transferOwnership(newProxyAdminOwner)
+}
+
+export default func
+
+func.tags = ["TransferProxyAdminOwnership"]
+func.dependencies = ["Bridge"]
```

Comment:
Should we [`func.skip = async () => process.env.GAS_REPORTER_BUG_WORKAROUND === "true"`](https://github.com/keep-network/keep-core/blob/5dd438efeeef4e5f71a264304f810786607f435e/solidity/ecdsa/deploy/09_transfer_proxy_admin_ownership.ts#L24-L27)?

Reply:
No, as we don't have the workaround here.